### ISSUE TITLE
Multi-line placement of widgets

### ISSUE THREAD

**Issue by Lansuper** [swttt/com.swttt.homeydash#57]
For I'm using a tablet in portrait mode, I would like the possibility to place widgets vertically on the dashboard. Now I can only change the order horizontally.
![win_20170930_08_28_13_pro](https://user-images.githubusercontent.com/32201989/31043282-9177322c-a5b9-11e7-8832-5241f727b14c.jpg)

**Comment by Mecallie** [comment]
Agreed. Also, the buienradar app is only one "bar" thick in portrait mode. You cannot make it a square, which ofcourse looks silly :)

Some portrait mode love would be nice!

**Comment by ArchiSpec** [comment]
Yes, please bring portrait functionality back. Screwed by betas since last stable v0.0.5. Tablets look much better on the wall in portrait mode but now broken.
